thetatauCMT/templates/events/event_form.html

```diff
@@ -8,10 +11,28 @@ <h1>Update Event: {{ event.name }}</h1>
   <form class="form-horizontal" method="post" action="{% url 'events:update' event.id %}">
     {% csrf_token %}
     {{ form|crispy }}
+    <h2>Pictures</h2>
```

Comment:
I thought we were not going to put pictures in the update form. Just in the main create form? Does this even work? I thought it didn't. If not, you should remove it.

Reply:
Viewing the pictures works but adding and removing pictures does not.